coderd/httpmw/workspaceagent.go

```diff
@@ -86,56 +87,23 @@ func ExtractWorkspaceAgent(opts ExtractWorkspaceAgentConfig) func(http.Handler)
 				}
 
 				httpapi.Write(ctx, rw, http.StatusInternalServerError, codersdk.Response{
-					Message: "Internal error fetching workspace agent.",
+					Message: "Internal error authorizing workspace agent.",
```

Comment:
```suggestion
					Message: "Internal error checking workspace agent authorization.",
```
This feels closer to what we're doing, but feel free to dismiss if you disagree.